### PR TITLE
Unarr port

### PR DESCRIPTION
**Pull request by miqlas**
'cause comics.

### CODE REVIEW

File: app-arch/unarr/unarr-1.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	sed -e s/-lm//g -i Makefile
+	make
```
Comment (waddlesplash) [inline app-arch/unarr/unarr-1.0.0.recipe]: $jobArgs?

File: app-arch/unarr/unarr-1.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	sed -e s/-lm//g -i Makefile
```
Comment (waddlesplash) [inline app-arch/unarr/unarr-1.0.0.recipe]: This belongs in the PATCH() function

**Comment by waddlesplash** [comment]
bump

**Comment by miqlas** [comment]
Waddlesplash, i'm ill as hell, sorry!

-- Z.

2016.03.06. dátummal, 2:15 időpontban waddlesplash <notifications@github.com> írta:

**Comment by miqlas** [comment]
Ready to rock!

File: app-arch/unarr/unarr-1.0.0.recipe [hunk 3]
```diff
@@ -0,0 +1,54 @@
+SUMMARY="Decompression library for RAR, TAR, ZIP and 7z archives"
+DESCRIPTION="
+unarr originated as a port of the RAR extraction features from \
```
Comment (diversys) [inline app-arch/unarr/unarr-1.0.0.recipe]: Should start from the previous line

File: app-arch/unarr/unarr-1.0.0.recipe [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp build/debug/unarr-test $binDir/unarr
+	mkdir -p $developDir
```
Comment (diversys) [inline app-arch/unarr/unarr-1.0.0.recipe]: Isn't it supposed to be in develop package?

**Comment by waddlesplash** [comment]
@miqlas Can you address @diversys' comments?

File: app-arch/unarr/unarr-1.0.0.recipe (hunk 4 above)
Comment (Begasus) [inline app-arch/unarr/unarr-1.0.0.recipe]: worked on the recipe a bit and uploaded a new one to my fork, maybe you could take a look at it?
https://github.com/Begasus/haikuports/commit/a0682402fb56055995f5b0a5dfc8fbee16b52a32#diff-d22ae810e98af09cafc5bd55e0af692d

**Comment by Begasus** [comment]
@miqlas ... maybe you could squash the work I've been doing on the pr or if you are ok with it I could create a new one?

**Comment by fbrosson** [comment]
@miqlas, here's a shorter link to @Begasus version: https://github.com/haikuports/haikuports/compare/master...Begasus:unarr
It'll also include any updates by him, in case he pushes other edits to his remote branch.

**Comment by miqlas** [comment]
I think i just let Begasus do it.

**Pull request closed without merging**